client/src/types/Settings.ts

```diff
@@ -2,6 +2,6 @@ import { Difficulty } from './Difficulty'
 
 export interface Settings {
   difficulty: Difficulty
-  roundsToWin: number
-  cardSpeed: number
+  roundsToWin: Number
```

Comment:
What's the difference? 🤔 

In the [Typescript doc](https://www.typescriptlang.org/docs/handbook/basic-types.html#number), they write it in lowercase.

Reply:
Hmm I think you're right, I can't remember why I changed it.